src/database/seeds/Participants.ts

```diff
@@ -78,6 +80,18 @@ export async function CreateParticipant(
     participantId: participant[0].id as number,
     participantTypeId: participantType[0].id as number,
   });
+
+  await AllowedSiteRoles[type].forEach(async (role) => {
```

Comment:
It smells a bit odd that a method in `api/services` is being called in a seed file. I think it might be better to just update the `sampleData` above to include the `apiRoles` manually. Or at least move the method to a more generic folder, maybe `api/helpers`. Or perhaps @lionell-pack-ttd can correct me if I'm off-base here?

Reply:
Yeah I think that's fair. I'm happy to basically just copy over the strings and use them in the loop instead?

Reply:
Seeds doesn't matter as much - I think it's OK to point to `api/services`, but it doesn't hurt to avoid the dependency as well.

It is important to avoid importing things to schema migrations. The migrations need to keep doing the same thing even as the code outside changes - if we need to change a table, it's a new migration. If we use code from somewhere like `api/services` in a migration, when that code changes the migration might change behaviour, and we wouldn't want that.